src/window/building_info.c

```diff
@@ -752,6 +772,12 @@ static void draw_foreground(void)
         } else if ((btype >= BUILDING_GRAND_TEMPLE_CERES && btype <= BUILDING_GRAND_TEMPLE_VENUS) ||
             btype == BUILDING_PANTHEON) {
             window_building_draw_grand_temple_foreground(&context);
+        } else if (building_monument_type_is_monument(btype)) {
```

Comment:
Because it's `else` from the conditional above it, it won't display for Grand Temples and Pantheon.

Reply:
Good point. this is not a mutually exclusive case.